scripts/include/common.sh

```diff
@@ -31,6 +31,12 @@ fi
 
 export DEEP_CLEAN="${DEEP_CLEAN:-false}" # If true, triggers helm to delete releases before cleaning up
 export KIND_VERSION="${KIND_VERSION:-0.2.1}"
+if [[ "$OSTYPE" == "darwin"* ]]; then 
+  export KIND_OS_TYPE="${KIND_OS_TYPE:-kind-darwin-amd64}" 
+else 
+  export KIND_OS_TYPE="${KIND_OS_TYPE:-kind-linux-amd64}" 
+fi
+
```

Comment:
I think it's better if you moved these to the file below (tools.sh). They are not needed anywhere else after all.